manifest_template.yml

```diff
@@ -46,25 +46,56 @@ apigee:
   - name: {{ ENV.name }}
     products:
       - name: {{ NAME }}
+        displayName: {{ TITLE }}
+        description: {{ DESCRIPTION }}
         approvalType: {{ approval_type }}
+        environments: [ {{ ENV.name }} ]
+        proxies:
+          - {{ SERVICE_NAME }}-{{ ENV.name }}
+          - identity-service-{{ ENV.name }}
         attributes:
           - name: access
             value: public
           - name: ratelimit
             value: {{ ENV.ratelimit | default('300pm') }}
+        scopes:
+          - 'urn:nhsd:apim:app:level3:{{ SERVICE_NAME }}'
+          - 'urn:nhsd:apim:user-nhs-login:P9:{{ SERVICE_NAME }}'
+          - 'urn:nhsd:apim:user-nhs-login:P5:{{ SERVICE_NAME }}'
+        quota: {{ ENV.quota | default('300') }}
+        quotaInterval: '1'
+        quotaTimeUnit: minute
+      - name: {{ NAME }}-user-restricted
+        displayName: {{ TITLE }} User-Restricted
         description: {{ DESCRIPTION }}
-        displayName: {{ TITLE }}
+        approvalType: {{ approval_type }}
         environments: [ {{ ENV.name }} ]
         proxies:
           - {{ SERVICE_NAME }}-{{ ENV.name }}
           - identity-service-{{ ENV.name }}
+        attributes:
+          - name: access
+            value: public
+          - name: ratelimit
+            value: {{ ENV.ratelimit | default('300pm') }}
         scopes:
           - 'urn:nhsd:apim:app:level3:{{ SERVICE_NAME }}'
```

Comment:
think this scope shouldn't be on user-restricted

Reply:
done